### ISSUE TITLE
Doesn't work on GOG version (Windows 10)

### ISSUE THREAD

**Issue by 3371-Alpha** [OrfeasZ/HitmanUnlocker#2]
The increase and decrease draw distance keybindings don't appear in the menus nor do they work if I bind them manually. I could use nemesis2000's fix but it only works on D3D; I prefer OpenGL as I can use it with reshade or enb. I've also tried switching modes to D3D but it didn't make you mod work.

**Comment by OrfeasZ** [comment]
Are you using the Steam version? This mod only works with the steam version of the game.

**Comment by 3371-Alpha** [comment]
No, is there a way to modify it to work with the GOG version? Both versions are identical except the stean version's exe is wrapped in their DRM.
